server/tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+	"compilerOptions": {
```

ensure you know every property and the reason here. Hekp me to find the reason of 
```suggestion
	    "target": "es2020",
		"module": "nodenext",
		"moduleResolution": "nodenext",
		"esModuleInterop": true,
		"skipLibCheck": true,
		"noEmitOnError": true,
		
```

---

I set "target": "es2020" because my server-side code uses modern JavaScript features like import/export syntax and the nullish coalescing operator (??). like:

is_public: wishlist.is_public ?? false

The nullish coalescing is only supported in es2020 and newer versions.